src/web/components/App.tsx

```diff
@@ -294,6 +299,37 @@ export const App = ({ CAPI, NAV }: Props) => {
 
 	const adTargeting: AdTargeting = buildAdTargeting(CAPI.config);
 
+	const YoutubeBlockComponent = loadable(
+		() => {
+			if (
+				CAPI.youtubeBlockElement.length > 0 ||
+				CAPI.youtubeMainMediaBlockElement.length > 0
+			) {
+				return import(
+					'@frontend/web/components/elements/YoutubeBlockComponent'
+				);
+			}
+			return Promise.reject();
+		},
+		{
+			resolveComponent: (module) => module.YoutubeBlockComponent,
+		},
+	);
+
+	const RichLinkComponent = loadable(
+		() => {
+			if (CAPI.richLinks.length > 0) {
+				return import(
+					'@frontend/web/components/elements/RichLinkComponent'
+				);
+			}
+			return Promise.reject();
+		},
+		{
+			resolveComponent: (module) => module.RichLinkComponent,
+		},
+	);
```

Comment:
Having this here, along with the other lazy loaded stuff, makes sense and the conditional is clear and easy to reason about 👍